### PR TITLE
capa Explorer Web: improve url navigation

### LINKED ISSUE #2419: explorer web: improve URL navigation

### PR DESCRIPTION
**Pull request by s-ff**
closes #2419 

This PR enhances the navigation in capa Explorer Web by making sure the links generated are shareable. [Preview](https://s-ff.github.io/capa/explorer/#/analysis?rdoc=https://raw.githubusercontent.com/mandiant/capa-testfiles/master/rd/al-khaser_x64.exe_.json).

Previously: users browsing to `/analysis` were always redirected to the homepage `/`.

With this PR:
- If a user accesses `/analysis` without an rdoc query parameter set in the URL, they are still redirected to the homepage.
- If a user accesses `/analysis` with the rdoc query parameter set, capa Explorer Web loads from that URL instead of redirecting to homepage.

### Checklist

- [ ] No CHANGELOG update needed

- [ ] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
 ### capa explorer IDA Pro plugin
 
 ### Development
+- capa Explorer Web: improve navigation in capa Explorer Web
```
Comment (mr-tz) [inline CHANGELOG.md:38]: let's add a new section `### capa Explorer Web` for these

**Review by mr-tz — APPROVED** [review]
looks good to me, thank you!

**Comment by mr-tz** [comment]
> TODO @s-ff: VirusTotal links have a short lifespan before expiration, add a toast message mentioning this instead of just "HTTP error 400"

will you create a separate PR for this?

**Comment by s-ff** [comment]
> > TODO @s-ff: VirusTotal links have a short lifespan before expiration, add a toast message mentioning this instead of just "HTTP error 400"
> 
> will you create a separate PR for this?

Yeah, I just created #2427 for tracking

File: CHANGELOG.md (hunk 1 above)
Comment (s-ff) [reply CHANGELOG.md]: 995d20ffb9199410c417bc4e72f5186e4f496609

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,9 @@
 
 ### capa explorer IDA Pro plugin
 
+### capa Explorer Web
+- improve navigation in capa Explorer Web @s-ff #2425
+
```
Comment (mr-tz) [inline CHANGELOG.md:39]:
```suggestion
### capa Explorer Web
- improve navigation in capa Explorer Web @s-ff #2425

### capa Explorer IDA Pro plugin
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -32,8 +32,10 @@
 - replace tabulate, tqdm, and termcolor with rich #2374 @s-ff
 - dynamic: emit complete features for A/W APIs #2409 @mike-hunhoff
 
-### capa explorer IDA Pro plugin
+### capa Explorer Web
+- improve navigation in capa Explorer Web @s-ff #2425
 
+### capa Explorer IDA Pro plugin
```
Comment (mr-tz) [inline CHANGELOG.md:38]:
```suggestion
### capa Explorer IDA Pro plugin

```

**Comment by mr-tz** [comment]
Thanks @s-ff!

**Pull request merged**